lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -2213,7 +2219,10 @@ mockProtocolParameters = dummyProtocolParameters
         , getTokenBundleMaxSize = TokenBundleMaxSize $ TxSize 4000
         , getMaxExecutionUnits = ExecutionUnits 10_000_000_000 14_000_000
         }
-    , minimumUTxO = MinimumUTxOConstant $ Coin 1000000
+    , minimumUTxO = minimumUTxOForShelleyBasedEra Cardano.ShelleyBasedEraBabbage
+        def
+            { Babbage._coinsPerUTxOByte = testParameter_coinsPerUTxOByte_Babbage
+            }
```

Comment:
Needed to update this to exactly match the values `Cardano.ProtocolParameters`. Opted not to do anything nicer now, but we should switch to have `balanceTx` take a `Ledger.PParams era` only, rather than the current `(W.ProtocolParameters, Cardano.ProtocolParameters)`.